Plug-Ins/Dialog Editor/DITLDocumentView.swift

```diff
@@ -0,0 +1,77 @@
+import Cocoa
+
+
+/// The "document area" of our scroll view, in which we show the DITL items.
+public class DITLDocumentView : NSView {
+	public override var isFlipped: Bool {
```

Comment:
It would be helpful if you could switch indentation to spaces to match RF's existing code style.